tests/aio-create.sh

```diff
@@ -214,15 +220,26 @@ pushd /opt/openstack-ansible
   # Disable tempest on older releases
   sed -i '/.*run-tempest.sh.*/d' scripts/gate-check-commit.sh  # Disable the tempest run
 
-  # NOTE(tonytan4ever): pin pip to the last working version before pip 10
+  # NOTE(tonytan4ever): pin pip to the last working version before pip 10	
```

Comment:
This change seems to be only adding whitespace?

Reply:
Oops, this can be gone.